pkgs/by-name/ri/ride/package.nix

```diff
@@ -33,8 +41,9 @@ let
 
   # Electron 27 is the latest version that works as of RIDE version 4.5.4097
   electron = electron_27;
+  electronDist = "${electron}/${if stdenv.isDarwin then "Applications" else "libexec/electron"}";
```

Comment:
```suggestion
  electronDist = electron + if stdenv.isDarwin then "Applications" else "libexec/electron";
```
is that valid nix?

Reply:
I think you meant something like this:
```nix
electronDist = electron + (if stdenv.isDarwin then "/Applications" else "/libexec/electron");
```
Not sure which one is better, though this new one doesn't use strings inside strings.

Ideally, this should be a property provided by the electron packages as `electron.dist`, but I haven't yet talked about this with the maintainers.